lib/manager/helmv3/artifacts.ts

```diff
@@ -60,9 +135,26 @@ export async function updateArtifacts({
     return null;
   }
   try {
+    // get repositories and registries defined in the package file
+    const packages = yaml.load(newPackageFileContent) as ChartDefinition; //TODO #9610
+    const locks = yaml.load(
+      existingLockFileContent.toString()
```

Comment:
Why toString? Should not be required.

Reply:
`existingLockFileContent` is a `Buffer` and the `load` function only accepts the content as string.

Reply:
Add utf-8 encoding to the `readLocalFile` then you'll get a string. 😉

Reply:
My thought has been that it is "safer" to compare the Buffers at https://github.com/renovatebot/renovate/pull/13726/files#diff-c2af1ba9c74f4a977e199d28c1e555ace1ce248ab708c0a5874fa5e5e2cd13c7R160, as encoding can not affect it then.     

But works for me.